setup.py

```diff
@@ -38,7 +38,7 @@
     ],
     'tests': tests_require,
     'kubernetes': [
-        'kubernetes>=6.0.0',
+        'kubernetes>=9.0.0',
```

Comment:
Please amend also `Werkzeug==0.14.1`  to use open-ended `>=` and please add the release commit